.github/pull_request_template.md

```diff
@@ -9,3 +9,29 @@
 - [ ] I have updated the documentation accordingly.
 - [ ] I have added tests to cover my changes.
 - [ ] All new and existing tests passed.
+
+## Description
+
+<!-- Please do not leave this blank -->
+
+## Related Issues
+<!-- 
+Please use this format link issue numbers: Fixes #123
+https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
+-->
+
+## Screenshots/Recordings
+
+<!--  screenshots  -->
+
+## Added tests?
+
+- [ ] yes
+- [ ] no, because they aren't needed
+
+## Added to documentation?
+
+- [ ] readme
+- [ ] contributing.md
+- [ ] docs
+- [ ] no documentation needed
```

Comment:
1. redme change to README
2. contributing is not needed
3. Docs change to Documentation
4. Change to No documentation Needed